webapp/client/public/css/components.css

```diff
@@ -294,8 +309,10 @@
     /* We need to still show it to keep focus on the element */
     height: 1px;
     width: 1px;
+    min-width: 1px;
```

Comment:
Similarly to #184 I do not understand why we need the min-width?

Reply:
We need to set it here, because we have to override the `min-width: 14px;` in the more general case.